server/src/main/java/io/seata/server/storage/db/lock/DataBaseDistributedLocker.java

```diff
@@ -251,4 +275,14 @@ private void init() {
         this.distributedLockDataSource = EnhancedServiceLoader.load(DataSourceProvider.class, datasourceType).provide();
     }
 
+    private boolean ignoreSQLException(SQLException exception) {
+        if (IGNORE_MYSQL_CODE.contains(exception.getErrorCode())) {
```

Comment:
return IGNORE_MYSQL_CODE.contains(exception.getErrorCode()) ||
           (StringUtils.isNotBlank(exception.getMessage()) && IGNORE_MYSQL_MESSAGE.stream().anyMatch(message -> exception.getMessage().contains(message)));

Reply:
Long bool judgment expressions are not recommended, as they become less readable.